### PR TITLE
Fix {error, disconnected} repainted with {error, notfound}

### PR DESCRIPTION
**Pull request by kuenishi**
In my environment, copying large object (say, 5GB) often caused 500 error to client, while some `{error, notfound}` left in log. This patch handles `disconnected` and `timeout` as simple retry, not asking for proxy-gets.

### CODE REVIEW

File: src/riak_cs_block_server.erl [hunk 1]
```diff
@@ -298,31 +298,33 @@ try_local_get(RcPid, FullBucket, FullKey, GetOptions1, GetOptions2,
...
+            _ = lager:debug("get_block_local/5 failed: {error, ~p}", [Why]),
+            RetryFun(NumRetries + 1);
+
+        {error, notfound} when UseProxyGet andalso ProxyActive->
+            case get_block_remote(RcPid, FullBucket, FullKey,
+                                  ClusterID, GetOptions2) of
+                {ok, _} = Success ->
+                    ProceedFun(Success);
+                {error, _} ->
+                    if UseProxyGet ->
```
Comment (shino) [inline src/riak_cs_block_server.erl]: `UseProxyGet` is always `true` here.

**Comment by kuenishi** [comment]
Fixed :)

**Comment by kuenishi** [comment]
TODO: update release notes

**Comment by kuenishi** [comment]
@borshop retry

**Comment by kuenishi** [comment]
+1 r=shino

**Comment by kuenishi** [comment]
@borshop merge

**Pull request merged**